src/views/shared/header/header.styles.ts

```diff
@@ -16,17 +16,34 @@ const useHeaderStyles = createUseStyles((theme: Theme) => ({
       margin: [theme.spacing(3), "auto", 0],
     },
   },
-  titleWrapper: {
+  topRow: {
     width: "100%",
-    padding: [theme.spacing(0.75), 0],
     display: "flex",
     alignItems: "center",
     justifyContent: "center",
   },
+  bottomRow: {
+    marginTop: theme.spacing(1.25),
+  },
+  leftBlock: {
+    flex: 1,
+    display: "flex",
+    justifyContent: "left",
+    gap: theme.spacing(0.75),
+  },
+  centerBlock: {
```

Comment:
You can remove it

Reply:
Yes, I know it has no effect but I prefer the explicitness.